docs/howto/flask-async.rst

```diff
@@ -0,0 +1,48 @@
+======================================================
+How to write a Kubernetes charm for an Async Flask app
+======================================================
+
+In this how to we will configure 12 Factor Flask application
+to use asynchronous Gunicorn workers to be able to serve
+to multiple users easily.
+
+Make the rock async
+===================
+
+Before packing the rock make sure to put the following in ``requirements.txt``
+file:
+
+.. literalinclude:: code/flask-async/requirements.txt
+
+Configure the async application
+===============================
+
+Now let's enable async Gunicorn workers using a configuration option. We will
+expect this configuration option to be available in the Flask app configuration
+under the keyword ``webserver-worker-class``. Verify that the new configuration
+has been added using
+``juju config flask-async-app | grep -A 6 webserver-worker-class:`` which should
+show the configuration option.
+
+.. note::
+
+    The ``grep`` command extracts a portion of the configuration to make
+    it easier to check whether the configuration option has been added.
+
+The worker class can be changed using Juju:
+
+.. literalinclude:: code/flask-async/task.yaml
+    :language: bash
+    :start-after: [docs:config-async]
+    :end-before: [docs:config-async-end]
+    :dedent: 2
+
+Now you can run
+``curl --parallel --parallel-immediate --resolve flask-async-app:80:127.0.0.1 \
+http://flask-async-app/io http://flask-async-app/io http://flask-async-app/io \
+http://flask-async-app/io http://flask-async-app/io``
+in they will all return at the same time.
```

Comment:
For my understanding, each instance of `http://flask-async-app/io` corresponds to a different async runner? Does "they will all return at the same time" mean that the async runners all respond to the `curl` request at the same time?

Perhaps it would be helpful to include example output here, although I don't know how universal the output is.